ironfish-cli/src/commands/wallet/send.ts

```diff
@@ -265,4 +278,44 @@ Find the transaction on https://explorer.ironfish.network/transaction/${
       this.exit(2)
     }
   }
+
+  async selectAsset(
+    client: RpcClient,
+    account: string | undefined,
+  ): Promise<string | undefined> {
+    const balancesResponse = await client.getAccountBalances({ account })
+    const assetOptions = []
+
+    // Get the asset name from the chain DB to populate the display choices
+    for (const { assetId } of balancesResponse.content.balances) {
+      const assetResponse = await client.getAsset({ id: assetId })
+
+      if (assetResponse.content.name) {
+        const displayName = BufferUtils.toHuman(Buffer.from(assetResponse.content.name, 'hex'))
+        assetOptions.push({
+          value: assetId,
+          name: `${assetId} (${displayName})`,
+        })
+      }
+    }
+
+    let assetId
+
+    if (balancesResponse.content.balances.length === 1) {
+      // If there's only one available asset, showing the choices is unnecessary
+      assetId = balancesResponse.content.balances[0].assetId
```

Comment:
We could check this condition before the loop above to avoid an unnecessary `getAsset` call

Reply:
I'm also thinking that in practice, if there's only one asset, doesn't it have to be $IRON in order to send a transaction? I don't think that we should hardcode that, but `send` might always fail if there's no iron for a tx fee

Reply:
That's a good point. I think we could improve the command's usability further with more guardrails. For example, the send command will happily let you start the command even if you have 0 balance, which can be confusing and strange. 

Also per the first comment, moved this up.